jina/proto/jina.proto

```diff
@@ -113,6 +113,8 @@ message Document {
 
     // List of evaluations performed on the document
     repeated Evaluation evaluations = 23;
+
+    bool indexed = 24;
```

Comment:
Since I guess this is internal fron jina and not suppoded to be exposed outside, we could use some reserved keys from tags like (__index__), we could do that with more things, for internal optimizations while hiding it from the external user

Reply:
agree, in fact not necessary to change protobuf, just directly write in tags `tags['is_indexed'] = True`

Reply:
Good point!